ntcore/src/generated/main/java/edu/wpi/first/networktables/BooleanArrayEntry.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
```

Comment:
Into all the generated files, we need to add a line to the end of the header block saying something like `// GENERATED BY <scriptname>`.

Reply:
`// THIS FILE WAS AUTO-GENERATED BY ./ntcore/generate_topics.py. DO NOT MODIFY`